WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.kt

```diff
@@ -481,6 +488,43 @@ class MySiteFragment : Fragment(),
         }
     }
 
+    private fun showTitleChangerDialog() {
+        // if site is self hosted we don't do anything on the tap
+        if (!SiteUtils.isAccessedViaWPComRest(selectedSite)) {
+            return
+        }
```

Comment:
Is there anything that would prevent us from making this check outside the click listener? That way we could disable the click altogether instead of just not doing anything — and the user wouldn't see the tap feedback.

Reply:
That's a good idea! We can't do it when setting up click listeners since site object might not be there yet, so I'm doing it in onResume, and it works ok :)